src/gameflow.h

```diff
@@ -43,7 +43,7 @@ class CGameflow
 
     uint8_t             getCurrentGameID();
     uint8_t             getCurrentLevelID();
-    const char*         getCurrentLevelName();
+    const char*         getCurrentLevelName(); ///@TODO: delete unused function!
```

Comment:
if it returns filename level name, it is duplication (maybe it is better to delete Engine_... version of that function); if it returns name like "Natla's mines" - it should be saved.